Sources/APNSKit/apns_kit.swift

```diff
@@ -1,3 +1,29 @@
+import NIOAPNS
+import NIO
+import NIOKit
+
+public final class APNSKitConnectionManager: ConnectionPoolSource {
+    public let eventLoop: EventLoop
+    private let config: APNSConfiguration
+    
+    deinit {
+        // Kill connection
```

Comment:
The connection pool will worry about closing connections when it is closed. No need to worry about that here.